ClinGen/clincoded
# Connects to #1775: Adds CAid to create by ClinVar VariationID

**Steps to test #1775**

1. Click the **New Variant Curation** button and enter any **ClinVar ID** in the variant lookup modal.
2. After clicking **Retrieve from ClinVar**, you should expect to see the **CA ID** listed
3. Click the **Save and View Evidence** button and in the green curation data section, under **Variant ID Sources**, you should expect to see the **CA ID** listed
